src/runtime_src/core/pcie/tools/xbmgmt/cmd_part.cpp

```diff
@@ -76,6 +76,8 @@ int program_prp(unsigned index, const std::string& xclbin, bool force)
         if (!errmsg.empty())
         {
             std::cout << errmsg << std::endl;
+            delete [] buffer;
```

Comment:
can we use smart pointer here for the buffer so that it'll be freed automatically?